khal/configwizard.py

```diff
@@ -113,7 +125,7 @@ def choose_time_format():
 def choose_default_calendar(vdirs):
     names = [name for name, _, _ in sorted(vdirs or ())]
     print("Which calendar do you want as a default calendar?")
-    print("(The default calendar is specified, when no calendar is specified.)")
+    print("(The default calendar is used as default, when no calendar is specified.)")
```

Comment:
`The default calendar is used as default` sounds a bit recursive. Like the dictionary definition:

Recursive: see recursive.

Reply:
my fault.